src/ioctls/system.rs

```diff
@@ -374,6 +376,46 @@ impl Kvm {
         }
     }
 
+    /// AArch64 specific function to create a VM fd using the KVM fd with flexible IPA size.
+    ///
+    /// See the arm64 section of KVM documentation for `KVM_CREATE_VM`.
+    /// A call to this function will also initialize the size of the vcpu mmap area using the
+    /// `KVM_GET_VCPU_MMAP_SIZE` ioctl.
```

Comment:
Can you also add an `# Arguments` section and specify what are the valid values for `ipa_size`?

From the kvm api documentation: " 32 <= N <= Host_IPA_Limit"

Reply:
Done.